lib/utils/bit_vector.ex

```diff
@@ -91,4 +91,9 @@ defmodule LambdaEthereumConsensus.Utils.BitVector do
     <<remaining::size(bit_size(bit_vector) - steps)-bitstring, _::bitstring>> = bit_vector
     <<0::size(steps), remaining::bitstring>>
   end
+
+  @spec to_byte(t) :: t
+  def to_byte(bit_vector) do
+    <<0::size(4), bit_vector::bits-size(4)>>
```

Comment:
This only works for 4-bit bitstrings

Reply:
should I add a check to see if the bit_vector is 4-bit? Or is there a better way?

Reply:
We should make it work on any bitstring, or make it a local helper function in the epoch processing module